analysis_options.yaml

```diff
@@ -19,10 +20,14 @@ linter:
     - iterable_contains_unrelated_type
     - list_remove_unrelated_type
     - non_constant_identifier_names
+    - omit_local_variable_types
     - overridden_fields
     - package_api_docs
     - package_names
     - package_prefixed_library_names
+    - prefer_relative_imports
+    - prefer_single_quotes
     - test_types_in_equals
     - throw_in_finally
+    - unawaited_futures
```

Comment:
this really didn't make it into the recommended set? cc @natebosch ?

Reply:
https://github.com/dart-lang/lints/issues/25